CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 The new options `system.aging-frequency` (default: 24 hours) and
```

Comment:
Nit: I would remove the parameterization from the changelog and focus rather on the description of the mechanism.

Reply:
As a user, I really like seeing new options listed in the changelog directly. I'll prepend another sentence explaining the feature itself more generally.

Reply:
I've improved the wording. Might taking a look again?